src/components/crossDevice/Intro/index.js

```diff
@@ -0,0 +1,46 @@
+import { h } from 'preact'
+import classNames from 'classnames'
+import theme from '../../Theme/style.css'
+import style from './style.css'
+import Title from '../../Title'
+import { trackComponent } from '../../../Tracker'
+import {preventDefaultOnClick} from '../../utils'
+
+const Intro = ({i18n, nextStep, componentsList, crossDeviceInitialStepType}) => {
+  const isFace = crossDeviceInitialStepType === 'face'
+  const steps = {
```

Comment:
since we call so many things steps, maybe `bulletPointsString` might be better?
(I actually don't know if they are bullet points)

Reply:
They're not, but I agree with renaming it